cm_sdk/client.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from cm_sdk.api import APIException
+
+class CloudManagerClient(object):
+    """
+    Simple client for interacting over HTTP with the the MongoDB
+    CloudManager API.
+    """
+
+    HEADERS = {'Content-type': 'application/json'}
+
+    def __init__(self, api_host, group_id, api_user, api_key):
```

Comment:
Not according to pylint ;).

I'm happy to name them, but I hardly feel 4 arguments are too many.